### ISSUE TITLE
Add a description how to setup eslint-plugin-vue on style guide page

### ISSUE THREAD

**Issue by re-fort** [vuejs/vuejs.org#1463]
Looking at the eslint-plugin-vue [demo](https://mysticatea.github.io/vue-eslint-demo/), it seems that the rules of priority A to C are already added.
I think it is about time to add a description how to setup eslint-plugin-vue on style guide page.
Or wait till the priority D rules have been added?

**Comment by chrisvfritz** [comment]
Yes, this is on my todo list! 😅 Priority D rules will possibly never be added to the ESLint plugin, since they're a bit more difficult to detect, but I do want to add links to the corresponding ESLint rules where they exist. 🙂

**Comment by jamessouth** [comment]
😆 I didn't know this existed so I published [vue-style-lint](https://github.com/jamessouth/vue-style-lint).  It only does one thing but I guess I'll stop working on it.
